packages/material-ui/src/styles/colorManipulator.js

```diff
@@ -27,14 +27,22 @@ function clamp(value, min = 0, max = 1) {
 export function hexToRgb(color) {
   color = color.substr(1);
 
-  const re = new RegExp(`.{1,${color.length / 3}}`, 'g');
+  const re = new RegExp(`.{1,${color.length >= 6 ? 2 : 1}}`, 'g');
   let colors = color.match(re);
 
   if (colors && colors[0].length === 1) {
     colors = colors.map((n) => n + n);
   }
 
-  return colors ? `rgb(${colors.map((n) => parseInt(n, 16)).join(', ')})` : '';
+  return colors
+    ? `rgb${colors.length === 4 ? 'a' : ''}(${colors
+        .map((n, index) => {
+          return index < 3
+            ? parseInt(n, 16)
+            : parseFloat(parseInt((parseInt(n, 16) / 255) * 1000, 10) / 1000);
```

Comment:
Would it be enough?

```suggestion
            : parseInt('f8', 16) / 255;
```

Reply:
I'm not sure. It can be long float parts.

Reply:
Which would result in higher accuracy?

Reply:
Yes, but browsers round numbers with big fractional part in different ways.

Reply:
Is your concern with the test case?

Reply:
Sorry, I don't understand.

Reply:
Ok, I see what you mean now. It wouldn't be a good option to display:

```css
color: rgba(212, 0, 52, 0.9725490196078431);
```